src/js/structs/Item.js

```diff
@@ -14,4 +14,8 @@ module.exports = class Item {
 
     return this._itemData[key];
   }
+
+  toJSON() {
+    return this._itemData;
```

Comment:
I agree, but there is another understanding: https://github.com/dcos/dcos-ui/pull/903#discussion_r73374936 The idea is to use the api instead of accessing the internal values.